keras/callbacks.py

```diff
@@ -985,23 +1005,62 @@ def on_epoch_end(self, epoch, logs=None):
 
 @keras_export("keras.callbacks.TerminateOnNaN")
 class TerminateOnNaN(Callback):
-    """Callback that terminates training when a NaN loss is encountered."""
+    """Callback that terminates training when a NaN loss is encountered.
 
-    def __init__(self):
+    Args:
```

Comment:
Let's be consistent with other classes and move the Args section to class doc

Reply:
it is already in the class doc